README.md

```diff
@@ -12,6 +12,7 @@ Check out the `examples/` folder for more examples. Here is `examples/clear/clea
 ```rust
 use sokol::app as sapp;
 use sokol::gfx as sg;
+use sokol::gfx as sglue;
```

Comment:
This should be `use Sokol::glue as glue;`, right?

Reply:
oops yep, thanks!

Reply:
Ok, fixed!